rush.json

```diff
@@ -11,8 +11,8 @@
 	},
 	"gitPolicy": {},
 	"repository": {
-		"url": "https://github.com/sedum-io/",
-		"defaultBranch": "develop",
+		"url": "https://github.com/Alchez/stonecrop",
```

Comment:
Can you change this to AgriTheory?

Reply:
I actually want to run some publishing tests on my fork first, I'll change it back when this is ready for review